packages/x-charts/src/LineChart/formatter.test.ts

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+import { FormatterParams } from '@mui/x-charts/models/seriesType/config';
+import lineFormatter from '@mui/x-charts/LineChart/formatter';
```

Comment:
These are private APIs, I think importing with a relative path could make more sense:
```suggestion
import { FormatterParams } from '../seriesType/config';
import lineFormatter from './formatter';
```

Reply:
Agree, I tried both to be sure the import wand not the cause of the issue